pkg/credhub/interpolate.go

```diff
@@ -0,0 +1,23 @@
+package credhub
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"code.cloudfoundry.org/buildpackapplifecycle/credhub"
+	"code.cloudfoundry.org/buildpackapplifecycle/platformoptions"
+	"code.cloudfoundry.org/goshims/osshim"
+)
+
+func InterpolateCredhubRefs(connectionAttempts int, retryDelay time.Duration) error {
+	if platformOptions, err := platformoptions.Get(os.Getenv("VCAP_PLATFORM_OPTIONS")); err != nil {
```

Comment:
Why is `"VCAP_PLATFORM_OPTIONS"` a parameter? Is it not always called with the same?

Reply:
When I reimplement the function I can make it not a parameter